uaclient/apt.py

```diff
@@ -455,16 +470,39 @@ def setup_apt_proxy(
     :param https_proxy: the url of the https proxy apt should use, or None
     :return: None
     """
+    scope = proxy_scope if proxy_scope else AptProxyScope.GLOBAL
     if http_proxy or https_proxy:
         event.info(messages.SETTING_SERVICE_PROXY.format(service="APT"))
```

Comment:
I think we can remove this line since we have the scope specific line.
```suggestion
```

Reply:
I see, yeah, the scoped message is complete in info.